statistics/cmsketch_test.go

```diff
@@ -49,12 +49,12 @@ func prepareCMSWithTopN(d, w int32, vals []*types.Datum, n uint32, total uint64)
 	return cms, nil
 }
 
-func buildCMSketchAndMap(d, w int32, seed int64, total, imax uint64, s float64) (*CMSketch, map[int64]uint32, error) {
+func buildCMSketchAndMapWithOffset(d, w int32, seed int64, total, imax uint64, s float64, offset int64) (*CMSketch, map[int64]uint32, error) {
```

Comment:
why extracting this function? Seems the only caller in the production code is `buildCMSketchAndMap()`.

Reply:
It is also used in line 233.

Reply:
got it